packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation_ios/FVPNativeVideoViewFactory.m

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "../video_player_avfoundation/include/video_player_avfoundation/FVPNativeVideoViewFactory.h"
+
+#import "../video_player_avfoundation/include/video_player_avfoundation/FVPNativeVideoView.h"
+#import "../video_player_avfoundation/include/video_player_avfoundation/FVPVideoPlayer.h"
+#import "../video_player_avfoundation/include/video_player_avfoundation/messages.g.h"
+
+@implementation FVPNativeVideoViewFactory {
+  NSObject<FlutterBinaryMessenger> *_messenger;
+  FVPVideoPlayer * (^_playerByIdProvider)(NSNumber *);
```

Comment:
The preferred repository style for new code is to use private properties and use `self.` access outside of init/dealloc.